source/minar.cpp

```diff
@@ -222,23 +222,23 @@ static minar::Scheduler* staticScheduler = NULL;
 /// - Implementation of minar class
 
 minar::Scheduler::CallbackAdder& minar::Scheduler::CallbackAdder::delay(
-    minar::tick_t delay
+    minar::tick_t delayIn
 ){
-    m_delay = delay;
+    m_delay = delayIn;
     return *this;
 }
 
 minar::Scheduler::CallbackAdder& minar::Scheduler::CallbackAdder::tolerance(
-    minar::tick_t tolerance
+    minar::tick_t toleranceIn
 ){
-    m_tolerance = tolerance;
+    m_tolerance = toleranceIn;
     return *this;
 }
 
 minar::Scheduler::CallbackAdder& minar::Scheduler::CallbackAdder::period(
-    minar::tick_t period
+    minar::tick_t periodIn
```

Comment:
the rest of this module uses camelCase only for function names, and underscore_separated local variables